Testing/VelaptorTests/Content/ContentLoaderTests.cs

```diff
@@ -87,92 +81,88 @@ public void Load_WhenLoadingFonts_LoadsFont()
             var loader = CreateContentLoader();
 
             // Act
-            loader.Load<IFont>(FontName);
+            loader.LoadFont(FontContentName, 12);
 
             // Assert
-            this.mockFontLoader.Verify(m => m.Load(FontName), Times.Once());
+            this.mockFontLoader.Verify(m => m.Load($"{FontContentName}|size:12"), Times.Once());
         }
 
         [Fact]
-        public void Load_WhenLoadingUnknownContent_ThrowsException()
+        public void Unload_WhenUnloadingTextures_UnloadsTexture()
         {
             // Arrange
-            var loader = CreateContentLoader();
+            var mockTexture = new Mock<ITexture>();
+            mockTexture.SetupGet(p => p.FilePath).Returns(TextureContentName);
 
-            // Act & Assert
-            Assert.ThrowsWithMessage<UnknownContentException>(() =>
-            {
-                loader.Load<IInvalidContent>("test-texture");
-            }, "Content of type 'VelaptorTests.Fakes.IInvalidContent' invalid.  Content types must inherit from interface 'IContent'.");
-        }
+            this.mockTextureLoader.Setup(m => m.Load(TextureContentName)).Returns(mockTexture.Object);
 
-        [Fact]
-        public void Unload_WhenUnloadingTextures_UnloadsTexture()
-        {
-            // Arrange
             var loader = CreateContentLoader();
-            loader.Load<ITexture>(TextureName);
+            var texture = loader.LoadTexture(TextureContentName);
 
             // Act
-            loader.Unload<ITexture>(TextureName);
+            loader.UnloadTexture(texture);
 
             // Assert
-            this.mockTextureLoader.Verify(m => m.Unload(TextureName), Times.Once());
+            this.mockTextureLoader.Verify(m => m.Unload(TextureContentName), Times.Once());
         }
 
         [Fact]
         public void Unload_WhenUnloadingSounds_UnloadsSound()
         {
             // Arrange
+            var mockSound = new Mock<ISound>();
+            mockSound.SetupGet(p => p.FilePath).Returns(SoundContentName);
+
+            this.mockSoundLoader.Setup(m => m.Load(SoundContentName)).Returns(mockSound.Object);
+
             var loader = CreateContentLoader();
-            loader.Load<ISound>(SoundName);
+            var sound = loader.LoadSound(SoundContentName);
 
             // Act
-            loader.Unload<ISound>(SoundName);
+            loader.UnloadSound(sound);
 
             // Assert
-            this.mockSoundLoader.Verify(m => m.Unload(SoundName), Times.Once());
+            this.mockSoundLoader.Verify(m => m.Unload(SoundContentName), Times.Once());
         }
 
         [Fact]
         public void Unload_WhenUnloadingAtlasData_UnloadsAtlasData()
```

Comment:
Refactor method name
```suggestion
        public void UnloadAtlas_WhenUnloadingAtlasData_UnloadsAtlasData()
```